addon/mixins/transition-mixin.js

```diff
@@ -26,203 +75,164 @@ export default Mixin.create({
     return this.get('transitionClasses').join(' ');
   }),
 
-  addClass(className, $element) {
+  addClass(className, element) {
     if (!this.get('isDestroying')) {
-      run(() => {
-        this.get('transitionClasses').addObject(className);
-      });
+      this.get('transitionClasses').addObject(className);
```

Comment:
https://github.com/peec/ember-css-transitions/pull/14

Other ppl were having the same problem.